src/pages/workspace/WorkspacesListPage.tsx

```diff
@@ -391,6 +404,35 @@ function WorkspacesListPage() {
         ],
     );
 
+    /**
+     * Gets the menu item for each domain
+     */
+    const getDomainMenuItem = useCallback(
+        ({item, index}: GetDomainMenuItem) => (
+            <OfflineWithFeedback
+                key={`domain_${item.title}_${index}`}
+                pendingAction={item.pendingAction}
+                style={styles.mb2}
+            >
+                <PressableWithoutFeedback
+                    role={CONST.ROLE.BUTTON}
+                    accessibilityLabel="row"
+                    style={[styles.mh5]}
```

Comment:
If it's just 1 style I would avoid passing it in an array. Direct object will suffice, otherwise we add overhead of flattening the array. Not a blocker, just a minor optimisation